derive_builder_core/src/setter.rs

```diff
@@ -151,7 +154,26 @@ impl<'a> ToTokens for Setter<'a> {
                         let mut new = #self_into_return_ty;
                         new.#field_ident = ::derive_builder::export::core::option::Option::Some(converted);
                         Ok(new)
-                }));
+                    }
+                ));
+            }
+
+            if let Some(ref ident_each) = self.each {
+                tokens.append_all(quote!(
+                    #(#attrs)*
+                    #[allow(unused_mut)]
+                    #vis fn #ident_each <ITEM>(#self_param, item: ITEM) -> #return_ty
+                    where
+                        #ty: ::derive_builder::export::core::default::Default + ::derive_builder::export::core::iter::Extend<ITEM>,
+                    {
+                        #deprecation_notes
+                        let mut new = #self_into_return_ty;
+                        new.#field_ident
+                            .get_or_insert_with(|| ::derive_builder::export::core::default::Default::default())
+                            .extend(::derive_builder::export::core::option::Option::Some(item));
```

Comment:
Is `Some` being used here instead of `iter::once`?

Reply:
Yes.

BTW, `iter::once` is implemented like this:

```
pub fn once<T>(value: T) -> Once<T> {
    Once { inner: Some(value).into_iter() }
}
```

:)

Reply:
Ideally, I would use `extend_one` here, but it is not stabilized yet. So, I just [inlined it](https://doc.rust-lang.org/stable/src/core/iter/traits/collect.rs.html#339-342).